WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -307,7 +307,7 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
   [locationManager startUpdatingLocation];
 
   CLAuthorizationStatus authStatus = [locationManager respondsToSelector:@selector(authorizationStatus)]
-    ? (CLAuthorizationStatus) [[locationManager performSelector:@selector(authorizationStatus)] integerValue]
+    ? (CLAuthorizationStatus) [locationManager performSelector:@selector(authorizationStatus)]
```

Comment:
Xcode is always showing a warning to that. Is there a way to avoid seeing it?

Reply:
Oh, which version? I often use Xcode 12.4 for now, but no warning here.

Reply:
Xcode 12.5-beta2

<img width="1267" alt="warn" src="https://user-images.githubusercontent.com/7767781/111577811-44513300-87b3-11eb-86d5-2c404832df7e.png">

Reply:
I assume we should use NSInvocation instead like https://stackoverflow.com/questions/23902132/perform-selector-casting advices

Reply:
66c6c9f seems good on Xcode 12.5 beta3